Orange/widgets/gui.py

```diff
@@ -1169,6 +1169,8 @@ def __getitem__(self, key):
                                     "S", (0, 0, 0)),
                                    (vartype(TimeVariable()),
                                     "T", (68, 170, 255)),
+                                   (vartype(Variable),
+                                    "?", (128, 128, 128)),
```

Comment:
But isn't `Variable` just a base class? Won't this open the door for certain errors to pass silently?